reduction_scripts/reduce_data.py

```diff
@@ -779,91 +882,103 @@ def run_save_3dcube(metadata, prev_suffix, curr_suffix, **args):
-        #------------------------------------------------------------------------
+        superbias_fn = "%s_superbias.fits" % calib_prefix
+        superbias_fit_fn = "%s_superbias_fit.fits" % calib_prefix
+        super_dflat_raw = "%s_super_domeflat_raw.fits" % calib_prefix
+        super_dflat_fn = "%s_super_domeflat.fits" % calib_prefix
+        super_dflat_mef = "%s_super_domeflat_mef.fits" % calib_prefix
+        super_tflat_raw = "%s_super_twiflat_raw.fits" % calib_prefix
+        super_tflat_fn = "%s_super_twiflat.fits" % calib_prefix
+        super_tflat_mef = "%s_super_twiflat_mef.fits" % calib_prefix
+        slitlet_def_fn = "%s_slitlet_defs.pkl" % calib_prefix
+        wsol_out_fn = "%s_wave_soln.fits" % calib_prefix
+        wire_out_fn = "%s_wire_soln.fits" % calib_prefix
+        flat_resp_fn = "%s_resp_mef.fits" % calib_prefix
+        calib_fn = "%s_calib.pkl" % calib_prefix
+        tellcorr_fn = "%s_tellcorr.pkl" % calib_prefix
+
+        # ------------------------------------------------------------------------
+        # ------------------------------------------------------------------------
         # RUN THE PROCESSING STEPS
         prev_suffix = None
         for step in proc_steps[arm]:
-            step_name   = step['step']
-            step_run    = step['run']
-            step_suffix = step['suffix']
-            step_args   = step['args']
-            func_name = 'run_'+step_name
+            step_name = step["step"]
+            step_run = step["run"]
+            step_suffix = step["suffix"]
+            step_args = step["args"]
+            func_name = "run_" + step_name
             func = locals()[func_name]
             if step_run:
-                func(obs_metadata,
-                    prev_suffix = prev_suffix,
-                    curr_suffix = step_suffix,
-                    **step_args)
+                func(
+                    obs_metadata,
+                    prev_suffix=prev_suffix,
+                    curr_suffix=step_suffix,
+                    **step_args,
+                )
                 if step_suffix != None:
                     prev_suffix = step_suffix
             else:
                 pass
 
-        #------------------------------------------------------------------------
-        #------------------------------------------------------------------------
+    # ----------------------------------------------------------
+    # Move reduce cube to the data_products directory
+
+    destination_dir = "./data_products/"
+
+    # Red
+    red_cubes_path = glob.glob("./data_products/intermediate/red/*.cube.fits")
+    for red_cube_path in red_cubes_path:
+        shutil.move(red_cube_path, destination_dir)
+
+    # Blue
+    blue_cubes_path = glob.glob("./data_products/intermediate/blue/*.cube.fits")
+    for blue_cube_path in blue_cubes_path:
+        shutil.move(blue_cube_path, destination_dir)
+
```

Comment:
The same for the lists `red_cubes_path` and `blue_cubes_path`, please use `working_dir`.